nat-lab/tests/test_pinging.py

```diff
@@ -149,7 +149,12 @@ async def test_session_keeper(
         )
 
         async def wait_for_conntracker() -> None:
+            counter = 0
             while True:
+                if counter % 60 == 0:
+                    print(f"{alpha_conn_tracker.get_out_of_limits()}")
+                    print(f"{beta_conn_tracker.get_out_of_limits()}")
+                counter += 1
```

Comment:
Could you clarify this change a little bit? Why it is needed?

Reply:
Some debug left, removing it now